distribution/lib/Standard/Base/0.0.0-dev/src/Internal/Instrumentor.enso

```diff
@@ -0,0 +1,92 @@
+private
+
+import project.Any.Any
+import project.Data.Text.Text
+import project.Function.Function
+import project.Meta
+import project.Internal.Meta_Helpers
+import project.Data.Vector.Vector
+import project.Nothing.Nothing
+import project.Runtime.Managed_Resource.Managed_Resource
+
+## PRIVATE
+   ADVANCED
+
+   Builder to create instrumentation for a function
+type Instrumentor
```

Comment:
`Instrumentor` is a nice concept, but probably not something we want to expose from a public API. It is still fine to have it and test it, but making `private` and moving the `Instrumentor_Spec` into `test/Base_Internal_Tests`.

- #11965

Y.js+Insight should be the future of instrumentation and it doesn't need the `Meta_Helpers.instrumentor_builtin`